app/src/main/dist/config/crate.yml

```diff
@@ -27,9 +27,13 @@
 # the owner to 'crate:crate'
 #path.data: /path/to/data1,/path/to/data2
 
-# Clustering: To avoid split-brain situations and data loss on recovery
-#gateway.expected_nodes: 5              # Total amount of nodes
-#gateway.recover_after_nodes: 3         # More than half of the nodes
+# Clustering: You can set the settings below to delay recovery until a
+# a number of data nodes are available to avoid the unnecesary creation of new 
```

Comment:
There are other ways to avoid replicas and also other reason, why one would want to wait for all nodes to be available. So I think these remark could be skipped, but am fine with keeping it.

Reply:
Maybe we could rephrase this, and use something along the context: "avoid to assign primary shards and replicas before a specified number of data nodes become available, as those shard allocations may be redistributed once more data nodes becomes available, unnecessarily increasing the work of the cluster" ?

This is somehow explained here: https://github.com/crate/crate/pull/13567/files#diff-3b72dd13af09151361214d2a5743f78ceb0a22ae65e19593fff105dedc515a71R885 and here: https://github.com/crate/crate/pull/13567/files#diff-c3e23f56fe5a6907288229d731cc2ca78629390997145fa79c57f0ce3a00e234R1336

Reply:
I rephrased this in https://github.com/crate/crate/pull/13567/commits/b0cc137dbf9f145ae855b26786d1d9fa20a70531